README.md

```diff
@@ -1,98 +1,120 @@
-# MongoDB Compass
-The MongoDB GUI.
-
-![Aggregation Pipeline Builder Tab in Compass](packages/compass/compass-screenshot.png)
-
```

Comment:
Is there a better screenshot we could add? Do we need one? I was thinking someone stumbling on this project through github might not know what Compass is and having a screenshot gives a quick understanding of how this project is a gui for interacting with a database (and also what they hopefully should see when they run the project).

Reply:
Yes, good point! I didn't remove it with some specific purpose, so totally will bring it back